app/assets/stylesheets/modules/footer.scss

```diff
@@ -0,0 +1,45 @@
+@import "variables";
+
+.footer {
+  background: $blue;
+  color: $white;
+  height: 300px;
+  position: absolute;
+  bottom: 0px;
+
+  .header {
+    font-family: Lato;
+    padding-top: 20px;
+    text-align: center;
+  }
+
+  h2 {
+    font-family: Lato;
+    line-height: 1.4;
+    color: $white;
+  }
+
+  hr {
+    border: solid #dddddd;
```

Comment:
Color literals like `#dddddd` should only be used in variable declarations; they should be referred to via variable everywhere else.